code/API_definitions/quality-on-demand.yaml

```diff
@@ -771,10 +771,10 @@ components:
       example: "123456789@domain.com"
 
     PhoneNumber:
-      description: A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, optionally prefixed with '+'.
+      description: A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, prefixed with '+'.
       type: string
-      pattern: '^\+?[0-9]{5,15}$'
-      example: "123456789"
+      pattern: '^\+[1-9][0-9]{4,14}$'
```

Comment:
Why is the number of digits following the +[1-9] reduced from 5-15 to 4-14?

This change is independent of requiring the +

Reply:
I looked at commonalties and this was also requiring too many digits for the minimum and allowing too many for the max.  So fixing two issues with the phone number regex.